tools/verify_partitions.py

```diff
@@ -0,0 +1,246 @@
+import argparse
```

Comment:
Now you're adding more and more scripts into `//tools`, we probably need to re-organize the layout for the scripts under `//tools` to have a clear hierarchy.

Reply:
As discussed in our late night call on Wednesday, we agreed that re-organizing the whole tests/tools directory is a time consuming task and should be done with due diligence. We agreed that this task can be taken-after the pipeline is made available for general public.